sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -63,16 +63,16 @@ export class BatchingReceiver extends MessageReceiver {
    * Receives a batch of messages from a ServiceBus Queue/Topic.
    * @param maxMessageCount The maximum number of messages to receive.
    * In Peeklock mode, this number is capped at 2047 due to constraints of the underlying buffer.
-   * @param idleTimeoutInSeconds The maximum wait time in seconds for which the Receiver
-   * should wait to receive the first message. If no message is received by this time,
-   * the returned promise gets resolved to an empty array.
+   * @param maxWaitTimeInSeconds The total wait time in seconds until which the receiver will attempt to receive specified number of messages.
+   * Once this time has elapsed the number of messages collected successfully in given time will be returned to the user.
```

Comment:
I would suggest the below change in the wording to reflect the relationship between the timer and the maxMessageCount

```suggestion
   * If this time elapses before the `maxMessageCount` is reached, then messages collected till then will be returned to the user.
```

Same for other places where the `maxWaitTimeInSecond` is used

Reply:
updated